modules/tripleliftBidAdapter.js

```diff
@@ -189,6 +189,11 @@ function _buildPostBody(bidRequests, bidderRequest) {
   if (bidderRequest?.ortb2?.regs?.gpp) {
     data.regs = Object.assign({}, bidderRequest.ortb2.regs);
   }
+
+  if (bidderRequest?.ortb2) {
+    data.ext.ortb2 = Object.assign({}, bidderRequest.ortb2);
```

Comment:
why take this here instead of just merging it in?

Reply:
@patmmccann sorry for the delay here. Just to maintain consistency throughout our adapter